cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -204,7 +204,8 @@ function Multiselect( element ) { // eslint-disable-line max-statements, inline-
 
     _optionsData.forEach( function( option ) {
       const _optionsItemDom = domCreate( 'li', {
-        'data-option': option.value
+        'data-option': option.value,
+        'class': 'm-form-field m-form-field__checkbox'
```

Comment:
Thanks for updating everything to the v4 classnames, I hadn't realized this hadn't been done.